src/site/content/en/blog/compat2021-midyear/index.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Compat2021 mid-year update: Flex gap everywhere"
```

Comment:
```suggestion
title: "Compat 2021 mid-year update: Flex gap everywhere"
```

I noticed we've been a bit inconsistent. I'm sending https://github.com/GoogleChrome/web.dev/pull/5871 and will suggest edits here.

Reply:
I was actually trying to be consistent by joining the year :D